packages/expo-manifests/ios/EXManifests/EXManifestsBaseManifest.h

```diff
@@ -18,6 +18,7 @@ NS_ASSUME_NONNULL_BEGIN
 - (nullable NSString *)slug;
 - (nullable NSString *)appKey;
 - (nullable NSString *)name;
+- (nullable NSString *)version;
```

Comment:
I've been backporting all changes to this module so far to the versioned stuff. We probably don't have to keep doing so but in this case they can be pretty easily backported (no versioned class prefix stuff).

Reply:
will follow up with that in a separate PR (maybe after SDK 43 versioned code lands)